Conch/cpanfile.snapshot

```diff
@@ -118,16 +118,18 @@ DISTRIBUTIONS
       ExtUtils::MakeMaker 0
       Scalar::Util 0
       perl 5.006
-  Class-C3-Componentised-1.001002
```

Comment:
This is rolling back module versions. Mind updating?

Reply:
Blew away `local` and ran carton install. No longer downgrading things. Pushed change.